Description: Patch in Debian provided jflex for generating java source
 during build.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: lucene3/common-build.xml
===================================================================
--- lucene3.orig/common-build.xml	2011-10-14 16:43:17.104113340 +0100
+++ lucene3/common-build.xml	2011-10-14 16:44:07.468499724 +0100
@@ -140,11 +140,8 @@
   <property name="jflex.home" location="${common.dir}"/>
 
   <path id="jflex.classpath">
-    <fileset dir="${jflex.home}/">
-      <!-- for a JFlex trunk checkout: -->
-      <include name="jflex/target/*.jar"/>
-      <!-- for a JFlex distribution (not yet available): -->
-      <include name="lib/*.jar"/>
+    <fileset dir="/usr/share/java">
+      <include name="JFlex.jar"/>
     </fileset>
   </path>
 
